### PR TITLE
[WIP] Install notes update

### PR DESCRIPTION
**Pull request by bollu**
I thought I should update the notes on `Installation` in the `README` to help newbies, so we could just point people to a set of pre-written steps

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -71,6 +71,14 @@ you to use the development version on Github (master branch). You need
 to clone the repository and install Vispy with
 ``python setup.py install``.
 
+As a one-liner, assuming `git` is installed: ::
```
Comment (Eric89GXL) [inline README.rst]: this can just be `is installed::`

**Comment by bollu** [comment]
can I add a "for contributors" section? That would contain stuff like how to setup vispy for development, info on `--vispy-fps`, `--vispy-log`, a link to the wiki, plus a link to point to all issues with the `for-new-contributors` tag. Does that sound okay?

**Comment by Eric89GXL** [comment]
Sure, but it should really probably go in `vispy-website` somewhere

**Comment by bollu** [comment]
Is there a subsection of the website specifically for developers?

**Comment by Eric89GXL** [comment]
No, not yet. But it would be nice to add a "contributing" section.

**Comment by bollu** [comment]
In that case, can this be merged? I'll make the other changes when I get time on the website.

**Comment by Eric89GXL** [comment]
Yeah, that will work

**Pull request merged**